integration/kubernetes/confidential/agent_image.bats

```diff
@@ -50,7 +50,7 @@ create_test_pod() {
 
 	echo "Create the test sandbox"
 	echo "Pod config is: $pod_config"
-	kubernetes_create_cc_pod $pod_config
```

Comment:
Hi @UnmeshDeodhar , I prefer the other way around: change the name of the functions that you are introducing on this PR to `Kubernetes_`, and leave the existing non-snp tests untouched. I want to avoid any regressions on the exsiting tests at this point  :)

Reply:
Changed all the functions back to `kubernetes_*`